src/main/java/frc/robot/commands/SpinRecordPlayer.java

```diff
@@ -21,16 +22,32 @@
 {
    private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
    private final RecordPlayerSubsystem powerRecordPlayer;
+   private final RobotContainer m_robotContainer;
    private final double m_motorPower;
-   public SpinRecordPlayer(RecordPlayerSubsystem subsystem, double motorPower)
+
```

Comment:
The spaces strike again...

Reply:
i'll have you know mi padre TOLD me to put those there. but whatever